syft/__init__.py

```diff
@@ -135,3 +135,12 @@ def pool():
     from syft.generic.id_provider import IdProvider
 
     ID_PROVIDER = IdProvider()
+
+import torch
+
+hook = TorchHook(torch)
+
+try:
+    opengrid = PublicGridNetwork(hook, "http://35.178.211.39:8080/")
```

Comment:
Could this be controlled by a flag or something? I don't think I want my PySyft install phoning the `PublicGridNetwork` every time it loads.